docs/first_steps.md

```diff
@@ -5,6 +5,7 @@ There are currently various sample simulations available to illustrate the use o
 
 * A uniform resolution volume run with GADGET and with SubFind halo catalogues - [click here](first_steps_gadget+subfind.md);
 * An alternative set of Rockstar halo catalogues based on the same GADGET simulation as above - [click here](first_steps_gadget+rockstar.md);
+* An alternative set of Rockstar halo catalogues based on the same GADGET simulation as above - [click here](first_steps_gadget+AHF.md);
```

Comment:
I assume this should say AHF halo catalogues...

Reply:
Ups, stupid copy and paste mistake... I will fix it.